test/inductor/test_max_autotune.py

```diff
@@ -185,6 +190,17 @@ def mm_plus_mm(a, b, c, d):
         ):
             torch.compile(mm_plus_mm)(a, b, c, d)
 
+    @parametrize("in_dtype_val", test_dtypes)
+    @parametrize("out_dtype_val", test_dtypes)
+    def test_max_autotune_matmul(self, in_dtype_val, out_dtype_val):
+        def quantized_matmul(x_vals, x_scales, w_vals):
+            return out_dtype(torch.ops.aten.mm.default, out_dtype_val, x_vals, w_vals) * x_scales
+        x_vals = torch.randn(65536, 144).to(dtype=in_dtype_val).cuda()
```

Comment:
`test_max_autotune` is shared by both CUDA and XPU，the hard code `cuda` here will break XPU, please decorated this test case with requires_cuda.

Reply:
LGTM